### PR TITLE
Add Default impl to RwLock

### PR DESCRIPTION
**Pull request by jorajeev**
Add Default impl to RwLock

---

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "shuttle"
-version = "0.0.1"
+version = "0.0.2"
```
Comment (jamesbornholt) [inline Cargo.toml:3]: Let's do this in a separate commit so we can start writing a changelog too.

Comment (jorajeev) [reply Cargo.toml]: Yeah, good point.  Resubmitting

**Review by jamesbornholt — APPROVED** [review]
